cypress/integration/todo_spec.js

```diff
@@ -55,13 +60,38 @@ describe('TodoMVC - React', () => {
   it('edits an existing todo', () => {
     addTodoItemInLocalStorage()
     todoItems()
-      .dblclick()
-      .find('.edit')
-      .clear()
-      .type(EDITED_ITEM)
-      .type('{enter}')
+    .dblclick()
+    .find('.edit')
+    .clear()
+    .type(EDITED_ITEM)
+    .type('{enter}')
 
     todoItems()
-      .should('contain', EDITED_ITEM)
+    .should('contain', EDITED_ITEM)
+  })
+
+  it('verify hint message', () => {
```

Comment:
I believe scenarios like `verify hint message` and `Verify logo text` should be part of a bigger picture. They are worthless by themself as they are not related to any business flow. But you could include these validations in scenario `it 'shows the form for adding todos'` or even `it 'adds new items'`